apps/e2e-tests/tests/frontend/terms-and-conditions.e2e-test.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+import { config } from '../../config';
+
+test('Should be able to see the terms-and-conditions page', async ({
+  page,
+}) => {
+  await page.goto(`${config.appUrl}/terms-and-conditions`);
```

Comment:
wouldn't it be better if we clicked the terms-and-conditions link that there is in the login page?

Reply:
i mean we could - but it does sound like adding bits to the scope. Do you think there is value in that? If there is i am happy to try, its best to make this more meaningful

Reply:
Could we reuse the typed routes from the routing package? The best end2end test follows the exact path a user might take but I already want to bypass authentication so atleast if we used the typed routings we would catch this reference if it was to ever change